### ISSUE TITLE
BOTW white screen and stucked

### ISSUE THREAD

**Issue by wu-xian** [Ryujinx/Ryujinx-Games-List#4317]
##### RAM: 16GB

crash when zelda first call link's name , sometimes white screen and stucked.
 
#### Log file:
[Ryujinx_ 1.1.0-macos1_2023-01-01_13-07-58.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/10328338/Ryujinx_.1.1.0-macos1_2023-01-01_13-07-58.log)

**Comment by wu-xian** [comment]
00:00:00.036 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.038 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.038 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.039 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.039 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.039 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.040 |I| Configuration LogValueChange: PreferredGpu set to: 0x106B_0xD0003EF
00:00:00.041 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.045 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.045 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.045 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.045 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.046 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.046 |I| Configuration LogValueChange: AudioVolume set to: 0.55
00:00:00.046 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.046 |I| Configuration LogValueChange: ExpandRam set to: True
00:00:00.046 |I| Configuration LogValueChange: IgnoreMissingServices set to: True
00:00:00.046 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.048 |N| Application PrintSystemInfo: Ryujinx Version:  1.1.0-macos1
00:00:00.051 |N| Application Print: Operating System: Darwin 22.1.0 Darwin Kernel Version 22.1.0: Sun Oct  9 20:15:09 PDT 2022; root:xnu-8792.41.9~2/RELEASE_ARM64_T6000 (Arm64)
00:00:00.051 |N| Application Print: CPU: Apple M1 Pro ; 8 logical
00:00:00.051 |N| Application Print: RAM: Total 16384 MiB ; Available Unknown
00:00:00.053 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.053 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:00.120 |I| Gpu : Backend Threading (Auto): True
00:00:00.425 |N| Application LoadGuestApplication: Using Firmware Version: 15.0.1
00:00:00.425 |I| Application LoadGuestApplication: Loading as NSP.
00:00:00.446 |I| Application AddAocItem: Found AddOnContent with TitleId 01007EF00011F001
00:00:00.446 |I| Application AddAocItem: Found AddOnContent with TitleId 01007EF00011F002
00:00:00.457 |I| ModLoader QueryContentsDir: Searching mods for Title 01007EF00011E000
00:00:00.458 |I| ModLoader QueryContentsDir: Searching mods for DLC 01007EF00011F001
00:00:00.458 |I| ModLoader QueryContentsDir: Searching mods for DLC 01007EF00011F002
00:00:00.459 |I| ModLoader QueryContentsDir: Searching mods for Title 01007EF00011E000
00:00:00.459 |I| ModLoader QueryContentsDir: Searching mods for DLC 01007EF00011F001
00:00:00.459 |I| ModLoader QueryContentsDir: Searching mods for DLC 01007EF00011F002
00:00:00.462 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:00.466 |I| Loader LoadExeFs: Loading rtld...
00:00:00.502 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.502 |I| Loader LoadExeFs: Loading main...
00:00:00.643 |I| Loader PrintRoSectionInfo: main:
    Module: D:\home\Cafe\U-King\trunk\Game\App\Rom\NX64\Product\code\U-King.nss
    SDK Libraries: SDK MW+Nintendo+NintendoWare_Atk-7_3_2-Release
                   SDK MW+Nintendo+NintendoWare_Font-7_3_2-Release
                   SDK MW+Nintendo+NintendoWare_G3d-7_3_2-Release
                   SDK MW+Nintendo+NintendoWare_Ui2d-7_3_2-Release
                   SDK MW+Nintendo+NintendoWare_Vfx-7_3_2-Release
                   SDK MW+Microsoft+Havok_Cloth-2014_2
                   SDK MW+Microsoft+Havok-2014_2
                   SDK MW+Microsoft+Havok_Animation-2014_2
                   SDK MW+Microsoft+Havok_AI-2014_2
                   SDK MW+Microsoft+Havok_Physics2012-2014_2
00:00:00.643 |I| Loader LoadExeFs: Loading subsdk0...
00:00:00.660 |I| Loader PrintRoSectionInfo: subsdk0:
    Module: multimedia
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_movie-7_3_2-Release
00:00:00.660 |I| Loader LoadExeFs: Loading sdk...
00:00:00.694 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 7.3.2
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libz-7_3_2-Release
                   SDK MW+Nintendo+NintendoSdk_nnSdk-7_3_2-Release
                   SDK MW+Nintendo+NintendoSDK_NVN-7_3_2-Release
00:00:00.724 |I| Ptc Initialize: Initializing Profiled Persistent Translation Cache (enabled: True).
00:00:00.742 |I| Loader LoadNsos: Loading image 0 at 0x0000000008000000...
00:00:00.743 |I| Loader LoadNsos: Loading image 1 at 0x0000000008004000...
00:00:00.750 |I| Loader LoadNsos: Loading image 2 at 0x000000000ad6a000...
00:00:00.751 |I| Loader LoadNsos: Loading image 3 at 0x000000000b487000...
00:00:00.755 |I| ModLoader LoadCheats: Build ids found for title 01007EF00011E000:
    39E10C07096696C9C4E228496C4BC44D6B9E0041000000000000000000000000
    8E9978D50BDD20B4C8395A106C27FFDE00000000000000000000000000000000
    93FF8252D975ADF07230CB86C83301C53F2804DF000000000000000000000000
    FCF93A41B08C5193A1410D262602DC621617C56F000000000000000000000000
00:00:00.755 |I| Loader LoadNca: Application Loaded: The Legend of Zelda: Breath of the Wild v1.6.0 [01007ef00011e000] [64-bit]
00:00:00.779 |I| GUI.WindowThread Hid Configure: Configured Controller Handheld to Handheld
00:00:00.779 |I| GUI.WindowThread Hid Configure: Configured Controller ProController to Player1
00:00:00.787 |I| GUI.WindowThread Hid SetupNpad: Connected Controller ProController to Player1
00:00:00.834 |S| HLE.OsThread.17 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:00:00.838 |S| HLE.OsThread.17 ServiceAm AcquireForegroundRights: Stubbed. 
00:00:00.838 |S| HLE.OsThread.17 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:00:00.838 |S| HLE.OsThread.17 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:00.846 |S| HLE.OsThread.17 ServicePctl Initialize: Stubbed. 
00:00:00.847 |N| GUI.RenderThread Gpu PrintGpuInformation: MoltenVK Apple M1 Pro (Vulkan v1.2.231, Driver v0.2.2008)
00:00:00.857 |I| GPU.MainThread Gpu LoadShaders: Loading 621 shaders from the cache...
00:00:01.505 |G| HLE.OsThread.17 ServiceLm Log: Guest Log:
  Log level: Info
    Time: 1s
    Line: 18
    Filename: 'D:/home/Cafe/U-King/trunk/Game/Lib/sead/engine/library/modules/src/basis/nin/seadRawPrintNin.cpp'
    Function: 'PrintStringImpl'
    Module: ''
    Thread: 'MainThread'
    ProgramName: 'U-King'
    Message: '0000002e'

00:00:01.508 |S| HLE.OsThread.17 ServiceAm SetRestartMessageEnabled: Stubbed. {restartMessageEnabled: True}
00:00:01.509 |S| HLE.OsThread.17 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:01.510 |S| HLE.OsThread.17 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:00:01.510 |S| HLE.OsThread.17 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:00:01.510 |S| HLE.OsThread.17 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:01.542 |S| HLE.OsThread.10 ServiceNv FinishInitialize: Stubbed. 
00:00:01.547 |I| HLE.OsThread.17 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:00:01.561 |S| HLE.OsThread.10 ServiceNv InitializeEx: Stubbed. 
00:00:01.571 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:01.600 |S| HLE.OsThread.10 ServiceNv SetNvMapFd: Stubbed. 
00:00:01.600 |S| HLE.OsThread.10 ServiceNv AllocGpfifoEx2: Stubbed. 
00:00:01.600 |S| HLE.OsThread.10 ServiceNv AllocObjCtx: Stubbed. 
00:00:01.601 |S| HLE.OsThread.10 ServiceNv SetTimeout: Stubbed. 
00:00:01.601 |S| HLE.OsThread.10 ServiceNv SetErrorNotifier: Stubbed. 
00:00:01.601 |S| HLE.OsThread.10 ServiceNv SetUserData: Stubbed. 
00:00:01.602 |S| HLE.OsThread.10 ServiceNv SetTimeslice: Stubbed. 
00:00:01.603 |S| HLE.OsThread.10 ServiceNv ZcullBind: Stubbed. 
00:00:01.634 |S| HLE.OsThread.9 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 2}
00:00:01.637 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8,Handheld
00:00:01.637 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 93 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight}
00:00:01.643 |S| HLE.OsThread.9 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075}
00:00:01.669 |S| HLE.OsThread.17 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72197163296350208}
00:00:01.671 |S| HLE.OsThread.17 ServiceAcc IsUserAccountSwitchLocked: Stubbed. 
00:00:01.690 |S| HLE.OsThread.17 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 1}
00:00:01.693 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=01007EF00011F001
00:00:01.695 |S| HLE.OsThread.17 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 1}
00:00:01.695 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=01007EF00011F001
00:00:02.293 |S| HLE.OsThread.10 ServiceNv ZbcSetTable: Stubbed. 
00:00:02.297 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:02.903 |S| HLE.OsThread.9 ServiceHid ActivateConsoleSixAxisSensor: Stubbed. {appletResourceUserId: 1}
00:00:02.903 |S| HLE.OsThread.9 ServiceHid InitializeSevenSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; counter0: 4096 ; counter1: 520192}
00:00:04.140 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:00:04.141 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV7
00:00:04.152 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:00:04.152 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:00:04.162 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 0
00:00:04.185 |I| AudioProcessor.Worker Audio EnsureAudioStreamSetup: New audio stream setup with a target sample count of 240
00:00:04.212 |S| HLE.OsThread.6 ServiceAudio QueryAudioDeviceSystemEvent: Stubbed. 
00:00:04.216 |S| HLE.OsThread.6 ServiceAudio GetActiveChannelCount: Stubbed. 
00:00:04.234 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 93 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair}
00:00:04.515 |S| HLE.OsThread.6 ServiceAudio GetActiveChannelCount: Stubbed. 
00:00:04.720 |S| HLE.OsThread.9 ServiceHid SetGyroscopeZeroDriftMode: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075 ; _gyroscopeZeroDriftMode: Tight}
00:00:04.721 |S| HLE.OsThread.17 ServiceAm SetGamePlayRecordingState: Stubbed. {_gamePlayRecordingState: True}
00:00:04.721 |S| HLE.OsThread.9 ServiceHid StopSevenSixAxisSensor: Stubbed. {appletResourceUserId: 1}
00:00:04.884 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:00:04.926 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:57.003 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:59.740 |I| HLE.GuestThread.16 Cpu PrintGuestStackTrace: Guest stack trace:
Process: Application, PID: 93
   0x00000000094f36f8	main:0x14ef6f8	
   0x00000000093c810c	main:0x13c410c	
   0x00000000093c7fec	main:0x13c3fec	
   0x00000000094cbdc0	main:0x14c7dc0	
   0x00000000094d9474	main:0x14d5474	
   0x00000000094cc28c	main:0x14c828c	
   0x00000000093ed350	main:0x13e9350	
   0x00000000092bfbbc	main:0x12bbbbc	
   0x00000000092c3910	main:0x12bf910	
   0x00000000092c4930	main:0x12c0930	
   0x00000000092c3f24	main:0x12bff24	
   0x00000000092c5020	main:0x12c1020	
   0x00000000093a4860	main:0x13a0860	
   0x00000000093a4404	main:0x13a0404	
   0x000000000950be7c	main:0x1507e7c	
   0x0000000009255718	main:0x1251718	
   0x0000000009255730	main:0x1251730	
   0x0000000009255730	main:0x1251730	
   0x0000000009255730	main:0x1251730	
   0x0000000009252e48	main:0x124ee48	
   0x0000000009618c60	main:0x1614c60	
   0x0000000009618950	main:0x1614950	
   0x0000000009254b2c	main:0x1250b2c	
   0x0000000008e5a2f4	main:0xe562f4	nnMain:0x2be4
   0x000000000b71dc9c	sdk:0x296c9c	nn::init::Start(unsigned long, unsigned long, void (*)(), void (*)()):0x004c
   0x00000000080000c0	rtld:0x00c0	


00:00:59.741 |I| HLE.GuestThread.16 Cpu PrintGuestRegisterPrintout: Guest CPU registers:
	X[00]:	0x0000000000000000
	X[01]:	0x00010010b8654559
	X[02]:	0x0000000000000000
	X[03]:	0x0000000000000000
	X[04]:	0x0000000000000001
	X[05]:	0x00000000094f3440 (main:0x14ef440)	=> 
	X[06]:	0x0000000000000000
	X[07]:	0x0000000000000001
	X[08]:	0x000000000b889610 (sdk:0x402610)	=> 
	X[09]:	0x00000000ffffffff
	X[10]:	0x00000000ffffffff
	X[11]:	0x0000000100000000
	X[12]:	0x00000002ae6b3700
	X[13]:	0x0000000000000058
	X[14]:	0x00000010a830c380
	X[15]:	0x00000010a830c380
	X[16]:	0x0000000000000020
	X[17]:	0x000000000b7aef80 (sdk:0x327f80)	=> nn::os::GetTlsValue(nn::os::TlsSlot)
	X[18]:	0x0000000000000380
	X[19]:	0x00000010ccb2a149
	X[20]:	0x0000000000000000
	X[21]:	0x000000118c301a48 (SP:-0x0108)
	X[22]:	0x0000000000000000
	X[23]:	0x0000000000000010
	X[24]:	0x00000010ccb2a101
	X[25]:	0x00000010a81bee88
	X[26]:	0x0000000000000000
	X[27]:	0x000000000abb8f68
	X[28]:	0x00000010a214df50
	FP:	0x000000118c301950 (SP:-0x0010)
	LR:	0x00000000094f36f8 (main:0x14ef6f8)	=> 
	SP:	0x000000118c301940 (SP)
	PC:	0x0000000092000006


00:00:59.741 |E| HLE.GuestThread.16 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x0000000000000000.
00:00:59.772 |E| HLE.GuestThread.16 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
   at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Memory/Tracking/MemoryTracking.cs:line 273
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.DataAbort(MemoryTracking tracking, UInt64 vcpu, UInt32 esr) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 227
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.SynchronousException(HvMemoryManager memoryManager, HvVcpu& vcpu) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 182
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.Execute(HvMemoryManager memoryManager, UInt64 address) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 160
   at Ryujinx.Cpu.AppleHv.HvCpuContext.Execute(IExecutionContext context, UInt64 address) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.Cpu/AppleHv/HvCpuContext.cs:line 30
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/ArmProcessContext.cs:line 40
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in /runner/_work/Ryujinx_Eve/Ryujinx_Eve/Ryujinx.HLE/HOS/Kernel/Threading/KThread.cs:line 1265
   at System.Threading.Thread.StartCallback()

**Comment by MetrosexualGarbodor** [comment]
Duplicate of #86
